Package.swift

```diff
@@ -1,5 +1,4 @@
-// swift-tools-version:6.0
-
+// swift-tools-version: 6.0
```

Comment:
This change should be reverted potentially

Reply:
Why?

Reply:
You added one missing space, I appreciate that 🙂